xml/iso_template.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+# This is the most basic code needed to load the template and write XML
+
+# From this directory, run: python iso_template.py
+
+
+from jinja2 import Environment, FileSystemLoader
+import yaml
+import re
+
+
+# eg if we have: {title_fr:"le title"}
+# get_alternate_text('title') will return [("fr", "le title")]
```

Comment:
Since we're using _eng for English, maybe similarly use the ISO _fra for French instead of _fr?

Reply:
I added a test so that an error is thrown if you try to set alternate language variables using the default language